test/scripts/Dockerfile.tests

```diff
@@ -1,4 +1,6 @@
-FROM node:latest
+FROM node:alpine
+
+RUN apk update && apk add curl
```

Comment:
there is `apk add --update curl` which should be more optimised, I read

Reply:
I'll leave the two commands because I couldn't find docs about the `--update` flag